### PR TITLE
Clarify vouching responsibilities

### PR DESCRIPTION
**Pull request by elavoie**
Hopefully closes issue #32 after refinements and a successful vote.

### CODE REVIEW

File: proposal-membership.md [hunk 1]
```diff
@@ -13,14 +13,23 @@
...
+
+After a new member has been accepted in the consortium, vouchers now share an ongoing responsibility towards the impact of that member on the consortium. This has both positive and negative implications. On the positive side, additional existing members may also decide to vouch for the new member: the more vouchers a member gets, the stronger others would like them to stay. The real weight of vouching however manifests when conflicts arise: vouchers then have a responsibility to help defuse and mediate conflicts, if the vouchee cannot or would not resolve them alone. If a voucher cannot or would not hold that responsibility, they should remove their vouching. In effect, vouching means: "I appreciate this person's contributions enough that I am willing to share the emotional burden of ensuring they can keep participating".
```
Comment (cblgh) [inline proposal-membership.md]: alternate phrasing for the last sentence, which i mentioned at the end of the meeting we had today

```suggestion
 After a new member has been accepted in the consortium, vouchers now share an ongoing responsibility towards the impact of that member on the consortium. This has both positive and negative implications. On the positive side, additional existing members may also decide to vouch for the new member: the more vouchers a member gets, the stronger others would like them to stay. The real weight of vouching however manifests when conflicts arise: vouchers then have a responsibility to help defuse and mediate conflicts, if the vouchee cannot or would not resolve them alone. If a voucher cannot or would not hold that responsibility, they should remove their vouching. In effect, vouching means:  "I trust this person and their abilities to participate fruitfully in the collective. I take on the responsibility of introducing the rules and spirit of the collective to them, and agree to intermediate if they are part of or cause conflicts."
```

Comment (elavoie) [reply proposal-membership.md]: @cblgh I actually replaced those paragraphs with only your sentence. I think it is a great summary and more upbeat in its formulation. See the latest revision to see my attempt in channeling you in being more sensitive and personal.

I will move the rest in the reasoning document.

**Comment by elavoie** [comment]
@cblgh @keks @staltz @ZELFs I reworked the how section, let me know if that suits you. I will move and rework the paragraphs I had initially written into the reasoning document.

**Comment by elavoie** [comment]
@cblgh @ZELFs I decided to use "caring for someone else" instead of "taking responsibility". I think it will set a more positive outlook on the underlying tasks and may better reflect your perspectives. I updated the reasoning document with two paragraphs that explain it. Let me know if I got your point of view right :-).

**Comment by cblgh** [comment]
this new revision feels a lot more approachable for someone new to the collective :) thanks for your thoughtful revisions @elavoie!

(also lol at the ssb collective using signal, cause ssb doesn't have private groups yet)

**Comment by elavoie** [comment]
@cblgh I lol'ed too... I considered mentioning that this was a temporary situation until we figure out private groups, but given the time it has been incubating I was wary of suggesting this might come soon. To our defense though, private groups based on cryptographic primitives in a fully decentralized setting *is* significantly more complicated and difficult than what I believe Signal implemented.

**Comment by arj03** [comment]
This needs to be rebased as it has conflicts and mentions consortium a few places towards the end.

**Comment by Powersource** [comment]
@arj03 could you do that? :)

File: proposal-membership.md [hunk 2]
```diff
@@ -8,19 +8,29 @@
...
 
+After fulfilling the criteria above, new members are added to a Signal group representing the collective. If members fail to maintain their active status, they are removed from the Signal group at the moment they become Inactive.
+
+### Vouching
+
+When you vouch for another member, it means: "I trust this person and their abilities to participate fruitfully in the collective. I take on the responsibility of introducing the rules and spirit of the collective, and agree to intermediate if they are part of or cause conflicts."
+
+Vouchings are public within the collective but private from non-members. For now, vouching relies on messages on Signal and your memory. Your responsibility in vouching also includes knowing who else vouches for the same person. Participate in the following ways:
+  1. **Add a vouch**: publicly message the other members of the Signal group mentioning who you are vouching for. This is also a good time to celebrate and publicly express the good things about your new vouchee!
```
Comment (staltz) [inline proposal-membership.md]: why "publicly"? Didn't the paragraph above say that vouching is "private from non-members"?

Comment (Powersource) [reply proposal-membership.md]: I think it's clear enough from the context that "publicly express etc" means publicly within the consortium

**Comment by Powersource** [comment]
Y

**Comment by Powersource** [comment]
Is this gonna be the first actual implementation of peer invites? :PPpppPPPpPpP

File: proposal-membership.md (hunk 1 above)
Comment (Powersource) [reply proposal-membership.md]: resolved?

File: proposal-membership.md (hunk 2 above)
Comment (staltz) [reply proposal-membership.md]: Still confusing IMO

**Pull request merged**